molecule/provisioner/ansible.py

```diff
@@ -248,15 +248,19 @@ class Ansible(base.Base):
 
     An alternative to the above is symlinking.  Molecule creates symlinks to
     the specified directory in the inventory directory.  This allows ansible to
-    converge utilzing its built in host/group_vars resolution.  These two
+    converge utilizing its built in host/group_vars resolution.  These two
     forms of inventory management are mutually exclusive.
+    Symlinking even allows to pass an additional inventory file (or dynamic inventory script),
```

Comment:
Could I nitpick on a new line above here for the purposes of readability?

Reply:
You can :-)

Reply:
Heroic